languages/ruby/exercises/concept/booleans/.docs/instructions.md

```diff
@@ -2,7 +2,7 @@ Continuing your work with the amusement park, you are tasked with writing some u
 
 ## 1. Check if an attendee has a ride pass
 
-Implement the `pass?` method to return a boolean (`true`/`false`) value based on the presence of a ride pass.
+Implement the `Attendee#pass?` method to return a boolean (`true`/`false`) value based on the presence of a ride pass.
```

Comment:
Does this need to return a true Boolean, or just a "boolean"?  If a true "Boolean" then capitalize the word (as it is a name) and leave the specfication as `true`/`false` there, as that matches.  If a "boolean" is indeed wanted, then remove the formal interpretation of that from the message.  It is confusing as it stands, if, as the reader of this, is aware of the difference.